awx/main/signals.py

```diff
@@ -72,40 +72,65 @@ def get_current_user_or_none():
     return u
 
 
-def emit_event_detail(serializer, relation, **kwargs):
+def emit_event_detail(cls, relation, **kwargs):
     instance = kwargs['instance']
     created = kwargs['created']
     if created:
-        event_serializer = serializer(instance)
```

Comment:
It turns out that this serializer has a *lot* of overhead, and we don't need or display the majority of the content it generates anyways (like related links and summary fields).

This change makes this code a bit more brittle, but this is a critically important section of our code from a performance perspective.  In my profiling, this change cut down per-event costs by over 50% 😱.